docs/introduction/open-source/public_code.md

```diff
@@ -92,108 +96,127 @@ Reviews MUST include source, policy, tests and documentation. | yes | repo is co
-There SHOULD be no acronyms, abbreviations, puns or legal/non-English/domain specific terms in the codebase without an explanation preceding it or a link to an explanation. | |Domain specific Dutch terms could be in a glossary which is also translated in English.
+Any translation MUST be up to date with the English version and vice versa. |  | TODO: add translations of user-facing texts (NL -> EN) (makemessages)
+There SHOULD be no acronyms, abbreviations, puns or legal/non-English/domain specific terms in the codebase without an explanation preceding it or a link to an explanation. |  | Domain specific Dutch terms could be in a glossary which is also translated in English.
 The name of the codebase SHOULD be descriptive and free from acronyms, abbreviations, puns or organizational branding. | yes |
-Documentation SHOULD aim for a lower secondary education reading level, as recommended by the [Web Content Accessibility Guidelines 2](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=315#readable). | | Would be good to get an evaluation of this prior to investing in translation.
-Any code, documentation or tests MAY have a translation. |
+Documentation SHOULD aim for a lower secondary education reading level, as recommended by the [Web Content Accessibility Guidelines 2](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=315#readable). |  | Would be good to get an evaluation of this prior to investing in translation.
+Providing a translation of any code, documentation or tests is OPTIONAL. |  |
 
 ## [Use open standards](https://standard.publiccode.net/criteria/open-standards.html)
 
-- [x] compliant with this criterion.
+- [x] criterion met.
 
 Requirement | meets | links and notes
 -----|-----|-----
-For features of a codebase that facilitate the exchange of data the codebase MUST use an open standard that meets the Open Source Initiative Open Standard Requirements. | yes | The [Zaakgericht Werken in het Gemeentelijk Gegevenslandschap](https://www.gemmaonline.nl/images/gemmaonline/f/f6/20190620_-_Zaakgericht_werken_in_het_Gemeentelijk_Gegevenslandschap_v101.pdf) meets the 5 criteria of the [Open Standards Requirement for Software](https://opensource.org/osr)
+For features of a codebase that facilitate the exchange of data the codebase MUST use an open standard that meets the [Open Source Initiative Open Standard Requirements](https://opensource.org/osr). | yes | The [Zaakgericht Werken in het Gemeentelijk Gegevenslandschap](https://www.gemmaonline.nl/images/gemmaonline/f/f6/20190620_-_Zaakgericht_werken_in_het_Gemeentelijk_Gegevenslandschap_v101.pdf) meets the 5 criteria of the [Open Standards Requirement for Software](https://opensource.org/osr)
+Any non-open standards used MUST be recorded clearly as such in the documentation. | n/a |
+Any standard chosen for use within the codebase MUST be listed in the documentation with a link to where it is available. | yes | <!-- link? -->
```

Comment:
I think this mostly comes down to upstream VNG deciding to standardize on OpenAPI 3